hadoop-ozone/interface-client/src/main/proto/OmClientProtocol.proto

```diff
@@ -1615,6 +1618,13 @@ message CreateSnapshotRequest {
   optional string snapshotName = 3;
 }
 
+message ListSnapshotRequest {
+  required string volumeName = 1;
+  required string bucketName = 2;
+  optional string startKey = 3;
```

Comment:
If you plan to provide richer API, we should have more parameters here e.g. endKey, Start-Creation-time, End-creation time etc.

Reply:
I'll make this pr simple first.